code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -347,3 +347,28 @@
 		to_chat(src, "<span class='critical'>Your regeneration is interrupted!</span>")
 		adjust_nutrition(-75)
 		active_regen = FALSE
+
+/mob/living/carbon/human/proc/setmonitor_state()
+	set name = "Set monitor display"
+	set desc = "Set your monitor display"
+	set category = "IC"
+	if(stat)
+		to_chat(src,"<span class='warning'>You must be awake and standing to perform this action!</span>")
+		return
+	var/obj/item/organ/external/head/E = organs_by_name[BP_HEAD]
+	var/datum/robolimb/robohead = all_robolimbs[E.model]
+	if(!E)
```

Comment:
You should enforce E (and E.model, don't know off the top of my head what happens when you index a list at null) _before_ you try and use it.

Reply:
oh right, i'll check that.

Reply:
i believe when you index a list at null or something it doens't has it just returns null, i'll move E.model below if(!E) but i think E = organs_by_name[BP_HEAD] if no head just gives back null. i'll check and update it.

Reply:
If it's an associative list, then yeah I think that's the behaviour. Numerical lists like to runtime when you try similar stuff though, because woooo consistency!